### ISSUE TITLE
Using external font forces reindex on every run

### ISSUE THREAD

**Issue by raxod502** [tectonic-typesetting/tectonic#118]
With sample document:

```
\documentclass{article}
\usepackage{fontspec}
\setmainfont{Georgia}[
Ligatures = TeX,
Path = /Library/Fonts/,
Extension = .ttf,
UprightFont = *,
BoldFont = * Bold,
ItalicFont = * Italic,
BoldItalicFont = * Bold Italic
]
\begin{document}
\textit{Hello!}
\end{document}
```

and fonts installed:

```
% ls -lA /Library/Fonts/Georgia*
-rw-r--r--  1 root  wheel  385768 Dec 12  2015 /Library/Fonts/Georgia Bold Italic.ttf
-rw-r--r--  1 root  wheel  363800 Dec 12  2015 /Library/Fonts/Georgia Bold.ttf
-rw-r--r--  1 root  wheel  400624 Dec 12  2015 /Library/Fonts/Georgia Italic.ttf
-rw-r--r--  1 root  wheel  379588 Dec 12  2015 /Library/Fonts/Georgia.ttf
```

I get the following on every run:

```
% tectonic -k document.tex
note: this is a BETA release; report issues at https://github.com/tectonic-typesetting/tectonic/issues
Running TeX ...
Running xdvipdfmx ...
note: indexing http://purl.org/net/pkgwpub/tectonic-default
note: resolved to https://dl.bintray.com/pkgw/tectonic/tl2016extras/2016.0r4/tlextras-2016.0r4.tar
note: downloading SHA256SUM
Writing document.pdf (11111 bytes)
Writing document.aux (8 bytes)
```

Note that this means Tectonic is hitting the network every time I compile my document, which is slow.

System information:

```
% neofetch
                -/+:.           raxod502@Radon-MBP-New.local
               :++++.           ----------------------------
              /+++/.            OS: OS X El Capitan 10.11.6 15G1510 x86_64
      .:-::- .+/:-``.::-        Model: MacBookPro11,3
   .:/++++++/::::/++++++/:`     Kernel: 15.6.0
 .:///////////////////////:`    Uptime: 29 days, 17 hours, 9 minutes
 ////////////////////////`      Packages: 353
-+++++++++++++++++++++++`       Shell: zsh 5.3.1
/++++++++++++++++++++++/        Resolution: 3008x1692@2x, 1440x900@2x
/sssssssssssssssssssssss.       DE: Aqua
:ssssssssssssssssssssssss-      WM: Quartz Compositor
 osssssssssssssssssssssssso/`   WM Theme: Blue
 `syyyyyyyyyyyyyyyyyyyyyyyy+`   Terminal: iTerm2
  `ossssssssssssssssssssss/     Terminal Font: InputMono-Medium 14
    :ooooooooooooooooooo+.      CPU: Intel i7-4980HQ (8) @ 2.80GHz
     `:+oo+/:-..-:/+o+/-        GPU: Intel Iris Pro, NVIDIA GeForce GT 750M
                                Memory: 5408MiB / 16384MiB
```

Tectonic installed from conda-forge:

```
% tectonic --version
Tectonic 0.1.5
```

**Comment by pkgw** [comment]
I *believe* that this will be fixed in the 0.1.6 release, which was just tagged yesterday. I'm still finishing the process of making packages and writing up the announcement, though.

Once you're able to get your hands on 0.1.6, can you try again and see if the problem persists?

**Comment by raxod502** [comment]
I can confirm that the problem is fixed in 0.1.6. Thanks!
